.github/workflows/test-full-sync.yml

```diff
@@ -5,7 +5,9 @@ on:
     inputs:
       network:
         default: 'Mainnet'
-  pull_request_review:
+  # only run on Mergify head branches:
+  # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-based-on-the-head-or-base-branch-of-a-pull-request-1
```

Comment:
```suggestion
```

Having the comment here might be confusing. Just inside the job should be enough to understand the `if` statement